geode-for-redis/src/main/java/org/apache/geode/redis/internal/netty/ByteToCommandDecoder.java

```diff
@@ -51,16 +53,16 @@
 public class ByteToCommandDecoder extends ByteToMessageDecoder {
 
   public static final String UNAUTHENTICATED_MAX_ARRAY_SIZE_PARAM =
-      "gemfire.geode-for-redis-unauthenticated-max-array-size";
+      RedisProperties.UNAUTHENTICATED_MAX_ARRAY_SIZE;
   public static final String UNAUTHENTICATED_MAX_BULK_STRING_LENGTH_PARAM =
-      "gemfire.geode-for-redis-unauthenticated-max-bulk-string-length";
+      RedisProperties.UNAUTHENTICATED_MAX_BULK_STRING_LENGTH;
 
   private static final int MAX_BULK_STRING_LENGTH = 512 * 1024 * 1024; // 512 MB
   // These 2 defaults are taken from native Redis
   public static final int UNAUTHENTICATED_MAX_ARRAY_SIZE =
-      Integer.getInteger(UNAUTHENTICATED_MAX_ARRAY_SIZE_PARAM, 10);
+      getIntegerSystemProperty(UNAUTHENTICATED_MAX_ARRAY_SIZE_PARAM, 10, 1);
   public static final int UNAUTHENTICATED_MAX_BULK_STRING_LENGTH =
-      Integer.getInteger(UNAUTHENTICATED_MAX_BULK_STRING_LENGTH_PARAM, 16384);
+      getIntegerSystemProperty(UNAUTHENTICATED_MAX_BULK_STRING_LENGTH_PARAM, 16384, 1);
```

Comment:
Could `UNAUTHENTICATED_MAX_ARRAY_SIZE_PARAM` and `UNAUTHENTICATED_MAX_BULK_STRING_LENGTH_PARAM` be replaced directly with `UNAUTHENTICATED_MAX_ARRAY_SIZE` and `UNAUTHENTICATED_MAX_BULK_STRING_LENGTH`? Having the same constant duplicated in two classes seems unnecessary.

Reply:
They could. At the time I didn't because they are public and I worried that they were used outside of the geode code base. But they probably are not so I'll try getting rid of them.